crates/driver/src/boundary/mod.rs

```diff
@@ -26,6 +26,7 @@ pub mod liquidity;
 pub mod mempool;
 pub mod quote;
 pub mod settlement;
+pub mod token_info;
```

Comment:
```suggestion
pub mod tokens;
```

I don't think `info` is particularly useful in the name.